### ISSUE TITLE
How to make the assistant write the first message?

### ISSUE THREAD

**Issue by simpel** [vercel/ai-chatbot#305]
Hi!

I'd like the assistant to start the conversation, basically just writing the first message. How can I do this so the thread understands it and treats it as the first message in the chat? Of course I can fake the UI so it looks like a message but the thread won't recognize it. 

This is basically what I want to achieve:
1. **Assistant**: Hi, my name is Mr. Assistant. I'm gonna mess up your career
2. **User**: Nice! What's your name?
3. **Assistant**: My name is Mr. Assistant

**Comment by intoxopox** [comment]
Yes, what's the best way to have the LLM start based on the instruction in the system prompt?
